src/Model/Command/Sorting.php

```diff
@@ -14,21 +14,40 @@ class Sorting extends AbstractCommand implements UserAwareInterface
     private $userId;
     /** @var string[] */
     private $itemIds = [];
+    /** @var string|null */
+    private $modelName = null;
 
-    private function __construct(string $userId, array $itemIds)
+    private function __construct(string $userId, array $itemIds, ?string $modelName = null)
```

Comment:
Maybe we can use `default` as default value and request `string`. As a benefit we can get rid of all null checks in other methods.

Reply:
On most clients, it will be - until they decide to use different model than default. `model_name` is optional parameter in the payload, allowing us to switch default models server-side.

For example, if the client decides to test default model against new model (`alpha`), and then decides to use the `alpha` as new default, then we won't *rename* the model -- we'd just switch configuration server side to use the new one for all requests that don't explicitly request a different model name.

Changing parameters of existing model needs full recalculation of the model - which is a costly operation. By the time the client decides to change the default model, we'll already have everything calculated & up to date in the new model; so doing a recalc feels kinda pointless if we can just change the default pointer server-side.